libc/include/llvm-libc-macros/features-macros.h

```diff
@@ -0,0 +1,15 @@
+//===-- Definition of macros from features.h
+//----------------------------------===//
```

Comment:
formatting

Reply:
github CI got angry at me for not doing this before :) Will try again